stdlib.sh

```diff
@@ -509,21 +509,19 @@ layout_anaconda() {
 # virtualenv from the Pipfile located in the same directory.
 #
 layout_pipenv() {
-  local venv
   PIPENV_PIPFILE="${PIPENV_PIPFILE:-Pipfile}"
   if [[ ! -f "$PIPENV_PIPFILE" ]]; then
     log_error "No Pipfile found.  Use \`pipenv\` to create a \`$PIPENV_PIPFILE\` first."
     exit 2
   fi
 
-  venv=$(pipenv --bare --venv 2>/dev/null)
```

Comment:
what does `--bare` do?

Reply:
Yeah, I hesitated removing that for any confusion. 😅

`--bare` prints ["Minimal output"](https://docs.pipenv.org/en/latest/#cmdoption-pipenv-bare), but `--venv` seems to not have any [conditional bare output](https://github.com/pypa/pipenv/blob/master/pipenv/cli/command.py#L147) (nor the code above) - either outputting only the path to stdout if exists or an error to stderr if it does not.

Since we previously only had `--bare` on the first `venv` definition (not the one that was actually exported) combined with the `[ ! d ...]` check, it seemed safe to remove. I'm happy to add it back though if you'd prefer.

Reply:
Okay, thanks for the explanation!